pkg/option/config.go

```diff
@@ -3362,6 +3388,15 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 		c.LogOpt = m
 	}
 
+	c.BPFEventsMapRateLimit = vp.GetUint32(BPFEventsMapRateLimit)
+	c.BPFEventsMapBurstLimit = vp.GetUint32(BPFEventsMapBurstLimit)
+	if c.BPFEventsMapRateLimit > 0 && c.BPFEventsMapBurstLimit == 0 {
+		c.BPFEventsMapBurstLimit = 3 * c.BPFEventsMapRateLimit
```

Comment:
I'm not a big fan of silently re-configuring flags for invalid configuration, it makes it difficult to understand the behaviour of Cilium in some cases.

Could we consider either doing `log.Fatal` or otherwise logging the reconfiguration as well as mentioning it in the flag usage string?